pkg/ccl/streamingccl/streamingest/stream_ingestion_processor.go

```diff
@@ -825,7 +825,7 @@ func (sip *streamIngestionProcessor) flush() (*jobspb.ResolvedSpans, error) {
 			sip.metrics.CommitLatency.RecordValue(timeutil.Since(minBatchMVCCTimestamp.GoTime()).Nanoseconds())
 			sip.metrics.Flushes.Inc(1)
 			sip.metrics.IngestedBytes.Inc(int64(totalSize))
```

Comment:
while we're here, would you mind renaming IngestedBytes to IngestedLogicalBytes and SSTBytes to IngestedPhysicalBytes? Or some other renaming scheme that makes this less confusing?